atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/JepsenHistoryChecker.java

```diff
@@ -42,6 +43,11 @@ public JepsenHistoryChecker(List<Checker> checkers) {
         return checkers;
     }
 
+    public static JepsenHistoryChecker createWithStandardCheckersForLock() {
+        return new JepsenHistoryChecker(
+                new LockCorrectnessChecker());
```

Comment:
nit: the constructors that accept arbitrary checkers as input are public - do we really need this method?